src/Button/index.tsx

```diff
@@ -0,0 +1,64 @@
+import type { FC, PropsWithChildren } from "react";
+
+import "./index.scss";
+import clsx from "clsx";
+
+export interface ButtonProps {
+  /**
+   * The button's size.
+   */
+  size?: "sm" | "lg";
+
+  /**
+   * The button's color scheme.
+   */
+  color?: "black" | "white" | "dark-blue" | "red" | "blue-a";
+
+  /**
+   * Whether the button is rounded.
+   */
+  rounded?: boolean;
+
+  /**
+   * Aria label for the button.
+   */
+  ariaLabel?: string;
+
+  /**
+   * Handler to fire when the button is clicked.
+   */
+  onClick(event: React.MouseEvent<HTMLButtonElement>): void;
+}
+
+/**
+ * A `<Button />` represents a standard clickable button. It serves as the base components for our many button types.
+ */
+const Button: FC<PropsWithChildren<ButtonProps>> = ({
+  children,
+  size = "sm",
+  color = "white",
+  rounded = false,
+  ariaLabel = "Button",
```

Comment:
Never have a default for accessible names. We want to omit this so that the button's accessible name can be determined from its child content, and passing `ariaLabel={undefined}` in React is the same as omitting it.